spec/iasm.dd

```diff
@@ -1067,6 +1067,16 @@ $(H3 $(LNAME2 amd_opcode_support, AMD Opcodes Supported))
         )
     )
 
+$(H3 $(LNAME2 Bit Manipulation Instructions, Bit manipulation instructions supported))
```

Comment:
three reasons:
- the big table is sorted alphabetically so it's unpleasant to insert something at the middle.
- these instructions are almost part of a special set called ABM1 and ABM2 (Advanced Bit Manipulation) which are, despite of not being specific to vectors, SSE2 only.
- I may propose more iasm ABM in a further PR.